rear_rider_device/camera_child_proc.py

```diff
@@ -1,48 +1,61 @@
+import os
 import asyncio
 from datetime import datetime
 from ipc.child_process import ChildProcess
-from typing import Deque
 
-from bluetooth_server_child_proc import BluetoothServerChildProcess
-
-import os 
 dir_path = os.path.dirname(os.path.realpath(__file__))
 
 class CameraChildProcess(ChildProcess):
-    def __init__(self, bt_server_proc: BluetoothServerChildProcess):
+    def __init__(self):
+        super().__init__(f'python {dir_path}/camera_proc.py')
         self.ready = asyncio.Future()
-        #self.bt_server_proc = bt_server_proc
+        self.start_time = datetime.now()
+
+
+    def _get_name(self) -> str:
+        return 'CameraChildProcess'
 
     async def on_ready(self):
         """
         An empty function.
 
-        An inheriting class can override this method to customize what happens when the child process is ready for communication
+        An inheriting class can override this method to customize what happens when the
+        child process is ready for communication
         """
         self.start_time = datetime.now()
         self._print('CameraChildProcess is ready.')
-        # try:
-            # await self.writeline('ready_ack')
+
         try:
             await self.writeline('get_stream')
-        except:
-            pass
+        except Exception as ex:
+            print(ex)
         self.ready.set_result(None)
         self._print('after_on_ready')
-    
+
 
     async def on_wait_ready(self):
         """
         An empty function.
 
-        An inheriting class can override this method to customize what happens before the parent process receives a ready signal from the child process.
+        An inheriting class can override this method to customize what happens before the
+        parent process receives a ready signal from the child process.
         """
         pass
-    
+
     def on_done(self):
         """
         An empty function.
 
-        An inheriting class can override this method to customize what happens when the child process is done communicating with the parent process. 
+        An inheriting class can override this method to customize what happens when the
+        child process is done communicating with the parent process.
         """
-        pass
\ No newline at end of file
+        pass
+
+    def no_on_handler(self, on_command, err):
+        """
+        An empty function.
+
+        An inheriting class can override this method to customize what happens when the
+        child process is done communicating with the parent process.
+        """
```

Comment:
This doc string is invalid for this method. I would suggest to remove it. The method is originally defined in `ChildProcess` but I didn't add a doc string. Let me know if you would like me to create a PR for updating the `ChildProcess` doc strings.

Reply:
The preview doesn't show it, but this is for the `no_on_handler` method, which is called when a does not have a handler for a specified message (on_command) type.

Reply:
yeah, we should update the `ChildProcess` doc strings too. I'll update this one to reflect what it's for 👍🏻